frontend/resources/src/OpenGL_Context.cpp

```diff
@@ -0,0 +1,17 @@
+#include "OpenGL_Context.h"
+
+#include "glad/gl.h"
+
+
+bool megamol::frontend_resources::OpenGL_Context::isVersionGEQ(int major, int minor) const {
+    if (version_ < GLAD_MAKE_VERSION(major, minor)) {
+        return false;
+    }
+    return true;
+}
+
+
+bool megamol::frontend_resources::OpenGL_Context::isExtAvailable(std::string const& ext) const {
+    auto const fit = std::find(ext_.begin(), ext_.end(), ext);
+    return fit != ext_.end();
+}
```

Comment:
Move whole OpenGL_Context.cpp to `frontend/services/opengl_glfw/`, the GL Service already links against glad and if there is no GL there is no `OpenGL_Context`

Reply:
Or make OpenGL_Context.h/OpenGL_Context.cpp not depend on glad by hiding relevant implementation details in the GL Service